docs/gallery/domain/images.py

```diff
@@ -0,0 +1,246 @@
+images to an :py:class:`~pynwb.file.NWBFile`.
+
+Image data can be of individual images or of movie segments (as a movie is simply a series of images),
+about the subject, the environment, the presented stimuli, or other kind
+that relates to the experiment.
+
+* :py:class:`~pynwb.image.OpticalSeries` for series of images that were presented as stimulus
+* :py:class:`~pynwb.image.ImageSeries`, for series of images (movie segments);
+* :py:class:`~pynwb.image.GrayscaleImage`, :py:class:`~pynwb.image.RGBImage`, :py:class:`~pynwb.image.RGBAImage`, for static images;
+
+The following examples will reference variables that may not be defined within the block they are used in. For
+clarity, we define them here:
+"""
+from datetime import datetime
+from dateutil import tz
+
+import numpy as np
+from PIL import Image
+
+from pynwb import NWBFile, NWBHDF5IO
+from pynwb.base import Images
+from pynwb.image import RGBAImage, RGBImage, GrayscaleImage, OpticalSeries, ImageSeries
+
+####################
+# Create an NWB File
+# ------------------
+#
+# Create an :py:class:`~pynwb.file.NWBFile` object with the required fields
+# (``session_description``, ``identifier``, ``session_start_time``) and additional metadata.
+
+session_start_time = datetime(2018, 4, 25, 2, 30, 3, tzinfo=tz.gettz("US/Pacific"))
+
+nwbfile = NWBFile(
+    session_description="Mouse exploring an open field",  # required
+    identifier="Mouse5_Day3",  # required
+    session_start_time=session_start_time,  # required
+    session_id="session_1234",  # optional
+    experimenter="My Name",  # optional
+    lab="My Lab Name",  # optional
+    institution="University of My Institution",  # optional
+    related_publications="DOI:10.1016/j.neuron.2016.12.011",  # optional
+)
+
+nwbfile
+
+####################
+#
+# .. seealso::
+#    You can learn more about the :py:class:`~pynwb.file.NWBFile` format in the :ref:`basics` tutorial.
+#
+# OpticalSeries: Storing series of images as stimuli
+# --------------------------------------------------
+#
+# :py:class:`~pynwb.image.OpticalSeries` is for series of images that were presented
+# to the subject as stimuli.
+# We will create an :py:class:`~pynwb.image.OpticalSeries` object with the name
+# ``"StimulusPresentation"`` representing what images were shown to the subject and at what times.
+#
+# Image data can be stored either in the HDF5 file or as an external image file.
+# For this tutorial, we will use fake image data with shape of (200, 50, 50, 3).
+# The second and third dimensions represent x and y.
+# The fourth dimension represents the RGB value (length of 3) for color images.
+#
+# NWB differentiates between acquired data and data that was presented as stimulus.
+# We can add it to the :py:class:`~pynwb.file.NWBFile` object as stimulus data using
+# the :py:meth:`~pynwb.file.NWBFile.add_stimulus` method.
+#
+
+image_data = np.random.uniform(low=0, high=255, size=(200, 50, 50, 3)).astype(np.uint8)
```

Comment:
should use [randint](https://numpy.org/doc/stable/reference/random/generated/numpy.random.randint.html#numpy.random.randint)